pumpkin/src/world/mod.rs

```diff
@@ -259,6 +259,22 @@ impl World {
         dbg!("DONE CHUNKS", inst.elapsed());
     }
 
+    /// Sends a message to all players
+    pub fn broadcast_message(&self, content: &TextComponent) {
+        for player in self.current_players.lock().values() {
+            player.send_system_message(content.clone());
+        }
+    }
+
+    /// Gets all players
+    pub fn get_players(&self) -> Vec<Arc<Player>> {
+        self.current_players
+            .lock()
+            .values()
+            .cloned()
+            .collect::<Vec<_>>()
+    }
```

Comment:
I don't like copying the Vec here

Reply:
Got it. Changed it to clone just the name string instead of the entire player